symphony/app/fbcnms-projects/magmalte/docker-compose.yml

```diff
@@ -29,7 +29,7 @@ services:
     build:
       context: ../..
       dockerfile: fbcnms-projects/magmalte/Dockerfile
-    command: "/usr/local/bin/wait-for-it.sh -s -t 30 mariadb:3306 -- yarn run start:dev"
+    command: "yarn run start:dev"
```

Comment:
this seems like a change in another PR

Reply:
I think I might need to just straight up ignore this file for commits as these were needed to get my localhost running, but probably don't need to be uploaded to the project as a whole.

What is the best way of removing this file from the PR?

Reply:
you mean you want it to be ignored and not included in the repo? If you don't want this to show up in the PR, you need to revert the change. I don't think we can put this in `.gitignore` as a file that doesn't get checked in

Reply:
@ekfuhrmann can you revert the changes to just this file and update the PR?